force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -172,7 +236,13 @@ public with sharing class FlowControlLoopsCollections {
         */
 
         // Return the final result string
-        return null; // Replace null with the variable you used to store the result
+        for (int i = 0; i < repeatCount; i++) {
+            result.append(inputString);
+            // If it is not the last iteration, add a semicolon to separate the strings
+        if (i < repeatCount - 1) {
+                result.append("; ");
+            }
+          return result.toString(); // Replace null with the variable you used to store the result }
     }
```

Comment:
Same thing. There Is no built-in StringBuilder class.  You just need to concatenate to a String